src/pathfinding_service/schema.sql

```diff
@@ -49,9 +49,12 @@ CREATE TABLE capacity_update (
     PRIMARY KEY (updating_participant, token_network_address, channel_id)
 );
 
-CREATE TABLE feedback_token (
+CREATE TABLE feedback (
     token_id CHAR(32) NOT NULL,
     creation_time TIMESTAMP NOT NULL,
     token_network_address CHAR(42) NOT NULL,
-    PRIMARY KEY (token_id)
+    route TEXT NOT NULL,
+    successful BOOLEAN CHECK (successful IN (0,1)),
+    feedback_time TIMESTAMP,
```

Comment:
Is this the time at which we received the feedback or the time it took to do the transfer? It sounds like the former, but IMO the latter would be more useful. Or do we assume `feedback_time - creation_time = transfer_duration`?

Reply:
This is the time the feedback is received, my thinking was that it's better to save the raw information, the duration is then easy to calculate from them.

Reply:
If we can easily calculate the duration, I agree. But it might get complicated if the first request does not succeed. The client might even wait for locks to expire before trying another route. But maybe I'm too pessimistic here and deriving the data will be fine.

Reply:
So you say we should take the latest feedback for a given token as the start, and the current time as the end?

Reply:
I think this is the only way to get timings out of this data. This will only work if the client tries each route immediately after the previous one, submits feedback immediately and the latency to the PFS is small compared to the duration of a payment.

That's why I thought making the client submit the duration explicitly would make data more robust and easier to handle.